_includes/cfp.html

```diff
@@ -0,0 +1,32 @@
+<!-- About Section -->
+<section id="about" class="about">
+    <div class="content-wrapper">
+        <div class="col-lg-10 col-lg-offset-1">
+            <h3>Call for contributions!</h3>
+            <p style="font-size: 20px;">
+                If you are working as research software engineer or are passionate about open-source (be it coding or community building), you are warmly encourage to present at the Nordic-RSE conference.
+
+                Relevant topics include but are not limited to
+                <ul>
+                    <li>High-performance computing</li>
+                    <li>Community building</li>
+                    <li>Teaching programming to researchers</li>
+                    <li>Developing research software</li>
+                    <li>Reproducible science</li>
+                    <li>Open-source, open-science, open access</li>
+                </ul>
+
+                At the conference, we offer different presentation formats
+
+                <ul>
+                    <li><b>Discussion session</b>: 1 hour session for an interactive discussion around a theme</li>
```

Comment:
it's not clear to me if we want to have this format. What do you think?

What about allowing to submit also a 3h workshop?

Reply:
Maybe rename to workshop which can also be a discussion session but sounds more prepared and less ad-hoc/diffuse?

Reply:
Now that we have also the `tutorial` as more structured 1h slot, maybe it's good this way?